crates/router/src/core/user/sample_data.rs

```diff
@@ -39,19 +39,21 @@ pub async fn generate_sample_data_for_user(
         .change_context(SampleDataError::InternalServerError)
         .attach_printable("Not able to fetch merchant key store")?; // If not able to fetch merchant key store for any reason, this should be an internal server error
 
-    let (payment_intents, payment_attempts, refunds): (
+    let (payment_intents, payment_attempts, refunds, disputes): (
         Vec<PaymentIntent>,
         Vec<PaymentAttemptBatchNew>,
         Vec<RefundNew>,
+        Vec<DisputeNew>,
```

Comment:
We can omit this part for now, let's cover only DB interface functions and queries, this we can cover in the next issue.